sdk/python/feast/online_response.py

```diff
@@ -57,7 +69,12 @@ def to_dict(self, include_event_timestamps: bool = False) -> Dict[str, Any]:
 
         for result in self.proto.results:
             for idx, feature_ref in enumerate(self.proto.metadata.feature_names.val):
-                native_type_value = feast_value_type_to_python_type(result.values[idx])
+                try:
+                    native_type_value = feast_value_type_to_python_type(result.values[idx])
+                except IndexError as e:
+                    logger.info(f'ERROR -- {e}')
+                    logger.info(f'ONLINE RESPONSE PROTO: {self.proto}')
```

Comment:
Shouldn't this be `DEBUB`? `INFO` could introduce a lot of latency.

Reply:
@woop  sorry this has just the purpose to show the source of error from the integration test [test_online_store_cleanup](https://github.com/feast-dev/feast/blob/master/sdk/python/tests/integration/online_store/test_universal_online.py) for `AWS:Redshift:dynamodb`.


> @adchia [test_online_store_cleanup](https://github.com/feast-dev/feast/blob/master/sdk/python/tests/integration/online_store/test_universal_online.py) with the case `Redshift` as offline store and `DynamoDB` as online store fails.
> 
> After doing some digging, there's a difference in the result of `fs.get_online_features` operation.
> 


Any thoughts why the results from `fs.get_online_features` are different?